available-trials.md

```diff
@@ -4,6 +4,7 @@ Sign up to enable any of these features on your origin by filling out [this form
 
 | Feature | Description | Timeline |
 | --- | --- | --- |
+| WebXR (For Chrome M67+) | Allows web sites to use virtual reality and applications that your audience can access on AR or VR enabled mobile devices or VR enabled headsets like Google's Daydream, Oculus Rift, Samsung Gear VR, HTC Vive, and Windows Mixed Reality Headsets. **This Origin Trial requires a different token than the existing WebVR trial.** | First available in Chrome M67. |
```

Comment:
General comment is we don't capitalize "Origin Trial" in normal sentences. Unless in a heading, lowercase is used (e.g. "origin trial" or "trial", similar to how "virtual reality" is earlier in the paragraph).

More specifically, I suggest emphasizing that new registration is required, rather than a different token. Also, I thought code changes and/or polyfill updates may be required? Maybe something like:
**This is a new trial that requires registration separately from the existing WebVR trial.**

Reply:
Done and done.

Reply:
PTAL